cmd/vc-rest/go.sum

```diff
@@ -117,12 +117,8 @@ github.com/hpcloud/tail v1.0.0 h1:nfCOvKYfkgYP8hkirhJocXT2+zOD8yUNjXaWfTlyFKI=
 github.com/hpcloud/tail v1.0.0/go.mod h1:ab1qPbhIpdTxEkNHXyeSf5vhxWSCs/tWer42PpOxQnU=
 github.com/hyperledger/aries-framework-go v0.1.3-0.20200408160748-24c4bd1cdda2 h1:bTMSNDpLs3qBU0yraX4/5REikVNyq1kP6XQJdQhJpwc=
 github.com/hyperledger/aries-framework-go v0.1.3-0.20200408160748-24c4bd1cdda2/go.mod h1:4SOyzXVQVIS/VWTEgMn4Li+xFRCWUSbUIF+ScN8JoxM=
-github.com/hyperledger/aries-framework-go v0.1.3-0.20200408183828-f12cbc1b8777 h1:W+jfwKZ2fDY/JzHXx/myC344Wznl/iLxbJz6JpdQ+YY=
-github.com/hyperledger/aries-framework-go v0.1.3-0.20200408183828-f12cbc1b8777/go.mod h1:4SOyzXVQVIS/VWTEgMn4Li+xFRCWUSbUIF+ScN8JoxM=
-github.com/hyperledger/aries-framework-go v0.1.3-0.20200408235155-7e4eafc4b66c h1:29UJcBcUpKQGaxmeh91CHc6QIRcsRy7lxYpXSGS6lQ0=
-github.com/hyperledger/aries-framework-go v0.1.3-0.20200408235155-7e4eafc4b66c/go.mod h1:4SOyzXVQVIS/VWTEgMn4Li+xFRCWUSbUIF+ScN8JoxM=
-github.com/hyperledger/aries-framework-go v0.1.3-0.20200411010800-c043ad2f2e92 h1:lhemfcJMWVh3jtY2EyJZp35lSKKkyvu7gcGFyC2PWI8=
-github.com/hyperledger/aries-framework-go v0.1.3-0.20200411010800-c043ad2f2e92/go.mod h1:4SOyzXVQVIS/VWTEgMn4Li+xFRCWUSbUIF+ScN8JoxM=
+github.com/hyperledger/aries-framework-go v0.1.3-0.20200413142855-397839f21ed4 h1:vLB7H+RL0hbcYOjWy+nhoyVG8a0TxzBo6+fGrU3tOHI=
+github.com/hyperledger/aries-framework-go v0.1.3-0.20200413142855-397839f21ed4/go.mod h1:4SOyzXVQVIS/VWTEgMn4Li+xFRCWUSbUIF+ScN8JoxM=
```

Comment:
I see a lot of older versions listed here. Perhaps try a `go mod tidy`?

Reply:
Those are removed in this PR. Am I missing something ?